### PR TITLE
Docs on how to set a custom directory for stylelib/

### PR DESCRIPTION
**Pull request by juanrg**
Just a phrase and the same ref as in the next section, as otherwise it is a bit confusing without understanding the internals of the library. Related to #6954

### CODE REVIEW

File: doc/users/customizing.rst [hunk 1]
```diff
@@ -32,7 +32,11 @@ path or URL to the style sheet. Additionally, if you add your
 your custom style sheet with a call to ``style.use(<style-name>)``. By default
 ``mpl_configdir`` should be ``~/.config/matplotlib``, but you can check where
 yours is with ``matplotlib.get_configdir()``; you may need to create this
-directory. Note that a custom style sheet in ``mpl_configdir/stylelib`` will
+directory. You also can change the directory where matplotlib looks for 
+the stylelib/ folder by setting the MPLCONFIG environment variable
+, see :ref:`locating-matplotlib-config-dir`.
```
Comment (QuLogic) [inline doc/users/customizing.rst]: Comma should be at the end of the word, not by itself here.

**Comment by NelleV** [comment]
Thanks for the contribution! Apart from @QuLogic's minor comment, it looks great.

File: doc/users/customizing.rst [hunk 2]
```diff
@@ -131,7 +135,7 @@ locations, in the following order:
 1. :file:`matplotlibrc` in the current working directory, usually used for
    specific customizations that you do not want to apply elsewhere.
 
-2. :file:`$MATPLOTLIBRC/matplotlibrc`.
```
Comment (tacaswell) [inline doc/users/customizing.rst]: I do not see how it grabs a file from `$MATPLOTLIBRC` (instead of being a directory) from looking at the source....

**Comment by juanrg** [comment]
Me neither. I don't know/remember where that line came from :/

**Pull request merged**